### PR TITLE
[#4036] Update the package name to use org.apache.gravitino 

### PR DESCRIPTION
**Pull request by rich7420**
### What changes were proposed in this pull request?

- Update the package name from com.datastrato.gravitino to org.apache.gravitino 

### Why are the changes needed?

- Repo moved.
Fix: #4036

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

`./gradlew build`

### CODE REVIEW

**Comment by jerryshao** [comment]
@yuqi1129 would you please help on this?

File: api/bin/test/log4j2.properties [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Name of the configuration
+name = ConsoleLogConfigDemo
+
+# Console appender configuration
+appender.console.type = Console
+appender.console.name = consoleLogger
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} %-5p [%t] %c{1}:%L - %m%n
+
+# Root logger level
```
Comment (yuqi1129) [inline api/bin/test/log4j2.properties:32]: Why do you add this file?

File: catalogs/catalog-hadoop/bin/test/log4j2.properties [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# Set to debug or trace if log4j initialization is failing
+status = info
```
Comment (yuqi1129) [inline catalogs/catalog-hadoop/bin/test/log4j2.properties:21]: ditto

Comment (yuqi1129) [reply catalogs/catalog-hadoop/bin/test/log4j2.properties:21]: The directory `bin` is unexpected in this PR

**Comment by yuqi1129** [comment]
The changes in this PR are too large to review, every time I get hung with the browser

**Comment by yuqi1129** [comment]
@jerryshao 
Please help to take a look.

**Comment by jerryshao** [comment]
Thanks @rich7420 for your work, thanks @yuqi1129 for your help. I'm going to merge this to avoid conflicts.

**Review by jerryshao — APPROVED** [review]

**Pull request merged**